apps/server/buffer.proto

```diff
@@ -58,6 +65,7 @@ message ClientRequest
   repeated bytes func_bytes_param = 40; // List of function parameters of type byte-array
   bool expect_reply = 50;               // Whether a response with data is expected or not
   FrameDetails frame_type = 60;         // Frame type information
+  SensorsInfo sensors_info = 70;        // List of informations regarding which sensor to address
```

Comment:
maybe reword in a simpler way: "The sensor to address" or "The sensor for which the request should apply"